test/safe/pull-request.test.ts

```diff
@@ -13,6 +13,14 @@ describe("GitHub Actions", () => {
 		app = await createWebhookApp();
 	});
 
+	afterEach(async () => {
+		if (!nock.isDone()) {
+			// eslint-disable-next-line jest/no-standalone-expect
+			expect(nock).toBeDone();
```

Comment:
Am I correctly understand, if nock is not done, then we call `expect(nock).toBeDone()` to fail the test?

Why not just use `expect(nock).toBeDone();` ?

Also should  
`nock.cleanAll()` be in a finally block? Hence if we fail the test on undone nock it won't be cleaned up